pom.xml

```diff
@@ -107,6 +103,9 @@ THE SOFTWARE.
     <changelog.url>https://jenkins-ci.org/changelog</changelog.url>
   </properties>
 
+  <!-- Note that the 'repositories' and 'pluginRepositories' blocks below are actually copy-pasted
+       from the Jenkins org pom. This is on purpose to keep jenkins core buildable even if one has
+       *not* defined the specific details in the settings.xml file. -->
```

Comment:
@olivergondza reverted the problematic changes, and added that comment to keep track of the discussion so that people in the future don't try and remove that w/o the considerations we had.

Thanks for the review